cms_form/static/src/js/ajax.js

```diff
@@ -0,0 +1,80 @@
+odoo.define('cms_form.ajax', function (require) {
+    'use strict';
+
+    var core = require('web.core'),
+        animation = require('website.content.snippets.animation');
+
+    animation.registry.CMSFormAjax = animation.Class.extend({
+        selector: ".cms_form_wrapper form[data-ajax]",
+        start: function () {
+            var data = this.$el.data('form');
+            this.$el.on('submit', this.proxy('submit_form'));
+            if (data.ajax_onchange) {
+                this.$el.on('change', this.proxy('submit_form'));
+            };
+            jQuery(data.form_content_selector).find(
```

Comment:
any reason not to use `this.$()` here and below?

Reply:
I sometimes have to work on code where `$` is some other hipster-JS-lib, since then I made it a habit to use the non-aliased name of libs

Reply:
but if you have 2 forms you are not 100% to select the right element. AFAIR `this.$` applies only in the scope of the current element.

Reply:
ah that's what you mean. Yes, this should be changed, I'm on it currently anyways